contrib/for-tests/resource-consumer/consume-cpu/consume_cpu.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"math"
+	"time"
+
+	"bitbucket.org/bertimus9/systemstat"
+)
+
+var d = time.Duration(10) * time.Millisecond
```

Comment:
Use a longer decriptive name and const, like:
const xxxYyyZzz =  time.Duration(10) * time.Millisecond